### ISSUE TITLE
Change l10n locale files

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/thimble.mozilla.org#2414]
tldr; split the `messages.properties` file into `client.properties` and `server.properties`

Currently, we have a single `messages.properties` file that has strings for both the client and server. However, there doesn't seem to be a single string that is used in both the client and server.

In the client, we have placeholders in the scripts for strings, for e.g. `{{ myStringKey }}`, which we replace with the string value (using nunjucks) during the build step and generate a localized version of the script file for each and every locale. This was a hack we did initially and its consequences are a slow l10n build step, a brittle file watching build step, and an error-prone template.

When I'm webpacking thimble, I want to change the way we localize the client so that the client loads strings dynamically (via a script tag) and only the strings it needs. For this, we need to be able to isolate the client strings from the `messages.properties` file. Since it's only 19 strings, we can separate it out of the `messages.properties` file into a `client.properties` file and rename the `messages.properties` file to `server.properties`. 

For now, I won't change the build step to dynamically load the strings (I'll do that in a webpack patch), but I will split the strings into the two files mentioned above, and change the localization script to account for that change.

cc @TheoChevalier

**Comment by gideonthomas** [comment]
Done in 404556174fa515d6b03edc0b6c36a9d39cb020ad a32fa85192bcfc5b2cbc633aa5ca8ec48a6908dc and ba695432dc344eb8fd0471086adcbfd2e4944653
